client/gutenberg/editor/init.js

```diff
@@ -26,7 +26,6 @@ const WPCOM_UNSUPPORTED_CORE_BLOCKS = [
 const loadA8CExtensions = () => {
 	// This will also load required TinyMCE plugins via Calypso's TinyMCE component
 	require( '../extensions/classic-block/editor' );
-	require( 'gutenberg/extensions/presets/jetpack/editor.js' );
```

Comment:
This is obsolete, since block registration/unregistration in Calypso is handled by `refreshRegistrations()` in the route handler.